src/api/controllers/articleController.js

```diff
@@ -7,7 +7,7 @@ import errorSender from '../../helpers/error.sender';
 import generatePagination from '../../helpers/generate.pagination.details';
 
 const {
-  Article, Category, User, Report
+  Article, Category, User, Report, Share
 } = models;
 
 const { CREATED } = statusCodes;
```

Comment:
can we use ``status`` instead of ``statusCodes``... destructuring doesn't change the meaning